.github/workflows/deploy-staging.yml

```diff
@@ -51,11 +51,15 @@ jobs:
         uses: ./.github/actions/install-ibmcloud-cli
       - name: Build project
         run: yarn build
-      - name: Building @carbon/ibmdotcom-web-components (staging) storybook
+      - name: Set env vars
+        uses: ./.github/actions/set-dotenv
+        with:
+          env-file: packages/web-components/.env
```

Comment:
@asudoh can you make the changes for the other environments like React too? I'm noticing the staging environments are not loading things like the CORS_PROXY, etc.

Reply:
@jeffchew Sure, done!